### ISSUE TITLE
Support for multiple feeds

### ISSUE THREAD

**Issue by DoCode** [loic-sharma/BaGet#155]
Like myget.
Under different URL's.

**Comment by LordMike** [comment]
Duplicate of #36  ?

**Comment by DoCode** [comment]
Nop, I come tomorrow with details for this feature.

**Comment by LordMike** [comment]
Ah, maybe #36 is mostly about auth on the service..

**Comment by loic-sharma** [comment]
This is a bit of a hack, but you can run two instances of BaGet side-by-side by setting the [`PathBase`](https://github.com/loic-sharma/BaGet/blob/master/src/BaGet.Core/Configuration/BaGetOptions.cs#L16) variable. These won't be able to share resources though. Is this acceptable?

**Comment by LordMike** [comment]
Maybe have a way to work in multi-tenancy in the web host?

It'd be great if feeds could be created from the UI, keys managed centrally -- one key for multiple feeds, etc.

**Comment by loic-sharma** [comment]
I'd like to do a prototype of multi tenancy on BaGet before committing to anything. I'm worried that this feature could hurt BaGet's ease-of-use and bring complexity to the code base.

**Comment by DoCode** [comment]
Yes, that's a different architecture, but when its injected in BaGet, it's irrelevant if a user has one or more feeds/tenants...

**Comment by kexxxfeng** [comment]
In despite  of the `PathBase` was set `/nuget/core`, the static files' href still were the root path `/static/css/main.a14e254f.css`

**Comment by loic-sharma** [comment]
It seems to be broken. I'll need to dig deeper into this, I wasn't able to figure this out easily.

**Comment by kexxxfeng** [comment]
@loic-sharma Have any news here? May I do something for you to resolve this issue?

**Comment by WernerMairl** [comment]
i'm working on a prototype for investigations about "multiple feeds"

routing can be enhanced with a few small changes, but i found one issue located here
![image](https://user-images.githubusercontent.com/10084630/54862390-4ace8080-4d3a-11e9-82f7-171e1e58d508.png)

the 2 red items are not rendering the tenant into the url "localhost/tenantid/v3/registration" because the call "url.AbsoluteUrl" is not supporting that, the other one using "url.AbsoluteRoutingUrl" are perfectly working=> i have a implementation proposal for that.....
but during regression tests i found one issue/question about trailing slash.

currently, "PackageBase" and "RegistrationBase" are delivered WITH trailing slash, the other items (PackagePublish, SymbolPublish etc) are delivered WITHOUT trailing slash.

Is this different behavior intended ?
if we change the implementation for the two red items into "AbsoluteRoutingUrl" then we deliver WITHOUT trailing slash, and i'm not sure about the impact....

Werner

**Comment by loic-sharma** [comment]
@kexxxfeng I haven't had a chance to look deeper into this yet.

@WernerMairl Whether there's a trailing slash shouldn't matter. There's no intention behind that behavior. What is `AbsoluteRoutingUrl`? I can't find documentation for that anywhere.

**Comment by WernerMairl** [comment]
AbsoluteRoutingUrl create a url based on the ROUTING information from the (incoming) request.
for my prototype i have added a additional "variable" into the routing template:
"http/localhost/{feed}/v3/{controller}"
AbsoluteRouteUrl ensures that the value for "{feed}" is resolved at runtime
